src/pages/AdminDashboard/ManageLocalAuthorities/ManageLocalAuthorities.module.scss

```diff
@@ -12,8 +12,13 @@
 
   thead {
     th {
-      background: g.$primary-mid-blue !important;
-      color: g.$primary-white !important;
+      background: g.$table-header-greyscale !important;
+      color: black !important;
```

Comment:
Is there a way of just adding better specificty rather than adding the important flag please? Not great practice

If this is for styling an antd component, you may have to add a new scss file for ant design overrides and import the classic way rather than how we do with modules. See src/components/JoinRequests or src/components/Dropdown for overrides examples

Reply:
Turns out the styles I added were actually the default, and what there was the override, so I just deleted it, and now its fine